bin/azure-deploy

```diff
@@ -105,6 +105,12 @@ while [ "$2" ]; do
   shift
 done
 
+if [ -z "$DOCKER_TAG" ]; then
+  echo "Usage: $0 ENVIRONMENT_NAME --docker-tag=DOCKER_TAG"
```

Comment:
I wonder if this is going to add an extra burden on developers to find the build number when running the script outside of the pipeline. Should we keep `development` / `production` / `test` tags for this purpose? Or perhaps introduce a release pipeline for test?

Reply:
Yeah, that's a good shout 👍